sqlglot/optimizer/qualify_columns.py

```diff
@@ -55,14 +55,13 @@ def qualify_columns(
     infer_schema = schema.empty if infer_schema is None else infer_schema
     dialect = Dialect.get_or_raise(schema.dialect)
     pseudocolumns = dialect.PSEUDOCOLUMNS
-
-    snowflake_or_oracle = dialect in ("oracle", "snowflake")
```

Comment:
Was this removed because `CONNECT BY` is only supported Snowflake & Oracle?

Reply:
yes